lib/elixir/test/elixir/kernel/lexical_tracker_test.exs

```diff
@@ -238,6 +238,22 @@ defmodule Kernel.LexicalTrackerTest do
 
       refute String in compile
       refute Enum in compile
+
+      {{compile, _, _, _}, _binding} =
+        Code.eval_string("""
+        defmodule Kernel.LexicalTrackerTest.Attribute7 do
+          Module.register_attribute(__MODULE__, :example, accumulate: true)
+          @example String
+          _ = Module.get_last_attribute(__MODULE__, :example)
```

Comment:
```suggestion
```

Reply:
This will better reveal the current behaviour.  But it is ok to return all (false positives are ok, false negatives are not).